pkg/kubectl/internal/podutils.go

```diff
@@ -0,0 +1,190 @@
+/*
```

Comment:
I think that we should move those to `pkg/kubectl/util` rather than this dir. I'm fine with us having the util as a throw-all bag for now. We'll deal with it later on.